definitions.yaml

```diff
@@ -658,3 +658,59 @@ definitions:
       auth_token:
         type: string
         description: The newly created API token
+
+  V4AppCatalogResponse:
+    type: array
+    items:
+      $ref: '#/definitions/V4AppCatalogItem'
+
+  V4AppCatalogItem:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Unique identifier for this app
```

Comment:
As this is supposed to be a unique identifier, I would go for `id` instead of `name`. Then `title` should be renamed to `name`. That would align it with the cluster resource.

Reply:
I do want to avoid an artificial ID on purpose if possible, we need to figure this still out though. In k8s this will ne a unique name as the name of a resource can only be unique (at least within a namespace), so maybe there's no need for generating additional unique IDs. I think it would be more beneficial to think about and comment on these things in the overarching design doc here: https://docs.google.com/document/d/1D2lQveGYbt2qd8nVJ8tW_0kCs6V0-62NrVxZhcyXPrY/edit?ts=5bc3ab5b#

Reply:
we do have indeed some issue/comment thread about how to do unique naming here that is simple and intuitive also for the time when a user might see it directly. also about how to keep versions apart (separate CRs vs list of versions inside CR)

Reply:
An ID doesn't have to be artificial. Like in the case of an organization.